src/api/v1/files.ts

```
@@ -0,0 +1,252 @@
+import path from "node:path";
+import db from "../../db/db";
+import { Jobs, Filename } from "../../db/types";
+import { authMiddleware } from "../middleware/auth";
+import { outputDir } from "../../index";
+import { ALLOW_UNAUTHENTICATED } from "../../helpers/env";
+
+export const files = new Elysia({ prefix: "/files" })
+  .use(authMiddleware)
+  .get(
+    "/:jobId/:fileName",
+    async ({ params: { jobId, fileName }, set, ...ctx }: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)